example/MNIST_cnn.jl

```diff
@@ -57,7 +57,7 @@ model =
     Flux.flatten,
     Dense(256 => 120, relu),
     Dense(120 => 84, relu),
-    Dense(84 => 10),
+    Dense(84 => 10),#TODO add softmax
```

Comment:
```suggestion
    Dense(84 => 10),
```
I would suggest creating issues for new features instead of todos